app/controllers/friendships_controller.rb

```diff
@@ -15,35 +15,36 @@ def sent
   end
 
   def create
-    receiver = User.find(params[:user][:receiver_id])
-    return if current_user.friend?(receiver)
-
-    Friendship.create(sender_id: current_user.id, receiver_id: receiver.id, accepted: nil)
+    friend = User.find(params[:user][:friend_id])
+    return if current_user.friend?(friend)
 
+    current_user.send_friend_request(friend)
+    # Friendship.create(friend_id: friend.id, user_id: current_user.id, accepted: nil )
```

Comment:
- [ ] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.